src/Modal.php

```diff
@@ -40,6 +40,7 @@ public function openModal($component, $componentAttributes = [], $modalAttribute
                 'dispatchCloseEvent' => $componentClass::dispatchCloseEvent(),
                 'destroyOnClose' => $componentClass::destroyOnClose(),
                 'maxWidth' => $componentClass::modalMaxWidth(),
+                'maxWidthClass' => $componentClass::modalMaxWidthClass(),
```

Comment:
For me, it doesn't work unless I change this line into:
```
$componentClass::modalMaxWidthClass($componentClass::modalMaxWidth()),
```

Otherwise, it will always be '2xl', is that correct?

Reply:
right !

Reply:
I was running the tests by changing the configs visually! I couldn't find a good way to dusk test this, but it lead me to miss this which kind of was the main intent of not locking it down! thats embarassing! 😬😳

Reply:
No worries! 🙌